### ISSUE TITLE
Human Review of the Paper

### ISSUE THREAD

**Issue by syifan** [syifan/ml-perf-survey#241]
There are many apparent problems in the paper. It is far from top-tier quality. I recommend you figure out what defines a top-tier conference and compare our paper against the standard. 

Here are some things I find as I read the paper. Please fix. 

1. Reduce abstract length
2. “We conduct hands-on reproducibility evaluations of five representative tools” — We need to evaluation all the tools that can be executed using a standard dataset that matches the potential user’s need. 
3. “reproducibility varies dramatically” — We probably do not evaluate reproducibility. Instead, we ignore their claim, but do our own evaluation. 
4. “score 8.5+/10 on our rubric” — I do not like having our own rubrics. We just break down and report details. We care most about performance and accuracy. 
5. Introuction — we need to better motivate. Also use shorter paragraphs. 
6. “The paper proceeds as follows:” Do not include this paragraph. 
7. Fig 1, text size too small. The text in the figure should be about the same size as main text. Consider using 2-column figure. 
8. Fig 1. What does the color coded to? 
9. If section 2 only has 2.1, then there is no need to have the section title. 
10. I do not understand the paragraph in 2.1. GPGPU-Sim is not a survey
11. “ML workloads, defined as computation graphs in frameworks like PyTorch [56] and TensorFlow [1], “  — Does not make sense, PyTorch and Tensorflows are frameworks. 
12. Is cycle-accurate method relevant here? Our topic is high-level simulations. 
13. 3.3 — Try only use fomulars if extremely necessary.
